cfme/utils/appliance/__init__.py

```diff
@@ -1542,6 +1542,45 @@ def update_rhel(self, *urls, **kwargs):
 
         return result
 
+    def upgrade(self, upgrade_to, cfme_only=True, reboot=False):
+        """Upgrade an appliance"""
```

Comment:
Can you add some information about upgrade paths? I.e. this is only for upgrading between z-streams (based off of L1550).